generic/tests/cfg/lvm.cfg

```diff
@@ -9,7 +9,7 @@
     image_format_stg1 = qcow2
     image_name_stg2 = images/storage_64k
     image_cluster_size_stg2 = 65536
-    image_size_stg2 = 1G
+    image_size_stg2 = 2G
```

Comment:
Thanks for the updating, that's totally what I was planed to do, but why you enlarge the image size here? is this necessary to avoid the issue?

Reply:
It is easy to check the disk driver in case have another issue.